cli/src/pcluster/config/cluster_config.py

```diff
@@ -2073,6 +2075,11 @@ def get_instance_types_data(self) -> dict:
         """Get instance type infos for all instance types used in the configuration file."""
         return {}
 
+    @property
+    def is_default_user_sudo_access_enabled(self):
+        """Return True if DisableSudoAccessForDefaultUser is True."""
+        return self.disable_sudo_access_default_user if self.disable_sudo_access_default_user else False
```

Comment:
Why `is_default_user_sudo_access_enabled` and `disable_sudo_access_default_user` have the same boolean value?